keymaps/intellij-idea-keymap.cson

```diff
@@ -230,6 +230,12 @@
   # Duplicate current line or selected block
   'ctrl-d': 'editor:duplicate-lines'
 
+  # Start new line
+  'shift-enter': 'editor:newline-below'
```

Comment:
Open an editor and type text on a new line "1234" 
Place your cursor between "2" and "3" 
With editor:newline, your result is:
"12\n"
"[cursor]34"
With editor:newline-below, your result is:
"1234\n"
"[cursor]" 

I really enjoy this feature.